code/datums/traits.dm

```diff
@@ -1030,3 +1030,67 @@ obj/trait/pilot
 	id = "atheist"
 	points = 0
 	isPositive = 0
+
+/obj/trait/lizard
+	name = "Reptilian (-1) \[Species\]"
+	cleanName = "Reptilian"
+	icon_state = "lizardT"
+	desc = "Trade in your human status and become a scaly, hissing lizardperson!"
+	id = "lizard"
+	points = -1
+	isPositive = 1
+	category = "species"
+
+	onAdd(var/mob/owner)
+		if(ishuman(owner))
+			var/mob/living/carbon/human/H = owner
+			H.set_mutantrace(/datum/mutantrace/lizard)
+		return
+
+/obj/trait/cow
+	name = "Bovine (-1) \[Species\]"
+	cleanName = "Bovine"
+	icon_state = "cowT"
+	desc = "Trade in your human status and become a furry, mooing cow!"
```

Comment:
Change this furry reference too.

Reply:
Expunged all reference to cows. And fur.